wiki/Skinning/Compendium/Anime/en.md

```diff
@@ -0,0 +1,208 @@
+[o!s]: /wiki/shared/mode/osu.png "osu!standard"
+[o!t]: /wiki/shared/mode/taiko.png "osu!taiko"
+[o!c]: /wiki/shared/mode/catch.png "osu!catch"
+[o!m]: /wiki/shared/mode/mania.png "osu!mania"
+
+# Anime
+Compendium of all completed anime osu! skins.
```

Comment:
Add a line between header and description for all.

Reply:
Is that something that is needed or just something thats suggested?